flake.nix

```diff
@@ -0,0 +1,24 @@
+{
+  description = "instantNIX";
+
+  nixConfig = {
+    extra-experimental-features = "nix-command flakes";
+
+    extra-substituters = [
+      "https://instantos.cachix.org"
+    ];
+
+    extra-trusted-public-keys = [
+      "instantos.cachix.org-1:6GezJZuxIK59ydzdyoIw6Yr4H6DaCoLzqhPaDhB8v24="
+    ];
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+  }: {
+    packages = import ./default.nix {
```

Comment:
People use it on many architectures, most notably Raspi. There's a library function that basically loops over the available aarchs, we should use that.